### DISCUSSION TITLE
list of files (images) in public folder

### DISCUSSION THREAD

**Discussion by edesideri** [quasarframework/quasar#10349] [General - Components / Directives / etc] [answered]
Hi,
how can i get the list of files (images) in public folder (public/images/..).
I would like to show all images with a v-for in a template.
thanks

**Comment by pdanpdan** [comment]
`ls` usually does a good job

**Reply by edesideri** [reply]
sorry maybe i dont understand.
how can i use ls command in a vue file to dinamically get the list of images in the public folder?

**Reply by pdanpdan** [reply]
You cannot. There is no way to get the list of files on a http server from the browser, except if there is an API on the server side to provide it to you.

**Comment by hawkeye64** [comment, marked as answer]
@edesideri You can do it at build time. Check out how QCalendar gets a file list from the "examples" folder so it can be used dynamically.
https://github.com/quasarframework/quasar-ui-qcalendar/blob/next/docs/src/assets/examples.js

Then it's imported here and used: https://github.com/quasarframework/quasar-ui-qcalendar/blob/next/docs/src/router/routes.js#L2

**Reply by edesideri** [reply]
Thanks a lot! it work greatly!

**Reply by s-light** [reply]
for vite based projects have a look at https://vite.dev/guide/features.html#glob-import
